dist/web3.js

```diff
@@ -1444,6 +1444,9 @@ web3.setProvider = function (provider) {
     this.currentProvider = provider;
     RequestManager.getInstance().setProvider(provider);
 };
+web3.isConnected = function(){
```

Comment:
Sure you do, but e.g. the wallet dapp will work in mist and normal browser (where it asks the user to start a node) and therefore a connection checker method is required and useful.
We can always later improve the way, how we check for connections:
```js
// set providor
if(!web3.currentProvider)
    web3.setProvider(new web3.providers.HttpProvider("http://localhost:8545"));
```